Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -85,6 +85,12 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
                         continue;
                     }
 
+                    // Let's skip collections that only have a private Setter
+                    if (property.SetMethod != null && property.SetMethod.IsPrivate)
```

Comment:
this change makes it so collection properties with private setters are NOT added to the mutable list.  Otherwise convention tests were failing saying that these properties werent obeying the Response object rules